APMrover2/mode.h

```diff
@@ -285,6 +285,9 @@ class ModeHold : public Mode
 
     // attributes for mavlink system status reporting
     bool attitude_stabilized() const override { return false; }
+
+    // hold mode does not require GPS
+    bool requires_gps() const override { return false; }
```

Comment:
I thought boats did station-holding in hold?

That code certainly isn't present now - was I imagining things?

Reply:
No, I don't think it ever did position hold in HOLD mode.  I was coincidentally thinking about the same thing earlier though - i.e. we need a position-hold mode for boats.